StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -422,7 +429,7 @@ extension EmbeddedPaymentElement {
             case .canceled:
                 let errorMessage = "confirm was called when the current update task is canceled. This shouldn't be possible; the current update task should only cancel if another task began."
                 stpAssertionFailure(errorMessage)
-                let error = PaymentSheetError.flowControllerConfirmFailed(message: errorMessage)
+                let error = PaymentSheetError.embeddedPaymentElementConfirmFailed(message: errorMessage)
```

Comment:
Can we use `unknown` here?  Or maybe a new unexpectedError case.

Reply:
Sure, did `unknown`